src/redux/books/books.js

```diff
@@ -1,4 +1,5 @@
-import BookstoreService from '../../services/BookstoreService';
+/* eslint-disable import/extensions */
+import BookstoreService from '../../services/BookstoreService.js';
```

Comment:
- [ ] The issue here was that you were importing this the wrong way. The correct import statement is `import BookstoreService from '../../services/bookstoreService';`. Kindly correct that because if you clone your project afresh it won't work due to the wrong import 👍